src/Sylius/Bundle/CoreBundle/Fixture/Factory/PromotionExampleFactory.php

```diff
@@ -170,4 +196,17 @@ private static function getCouponNormalizer(?FactoryInterface $couponFactory): \
             return $coupons;
         };
     }
+
+    private function getLocales(): iterable
+    {
+        if (null === $this->localeRepository) {
+            throw new \RuntimeException('You must configure a $localeRepository');
+        }
```

Comment:
We should return here an empty array to maintain the previous behaviour and not to provide a BC break